axelor-quality/src/main/resources/i18n/messages_fr.csv

```diff
@@ -132,9 +148,14 @@
 "Start date","Date de début",,
 "Status","Statut",,
 "Status select",,,
+"Status with same priority already exists.","Il existe déjà un statut avec une priorité identique.",,
+"Supplier","Fournisseur",,
+"System",,,
 "Tag color","Couleur étiquette",,
 "Tags","Etiquettes",,
 "Team","Equipe",,
+"Technical sequence","Séquence technique",,
+"The company %s doesn't have any configured sequence for Quality improvement",,,

```

Comment:
Translation can be: "La société %s n'a pas de séquence configurée pour Amélioration qualité"